modules/administration/pages/live-patching-sles12.adoc

```diff
@@ -0,0 +1,113 @@
+[[live-patching-sles12]]
+= Live Patching on {slsa}{nbsp}12
+
+
+On {slsa}{nbsp}12 systems, live patching is managed by kGraft.
+For in depth information covering kGraft use, see https://www.suse.com/documentation/sles-12/singlehtml/book_sle_admin/book_sle_admin.html#cha.kgraft.
+
+Before you begin, ensure:
+
+* {productname} is fully updated
+* You have one or more Salt clients running {slsa}{nbsp}12 (SP1 or later)
+* Your {slsa}{nbsp}12 Salt clients are registered with {productname}
+* You have access to the {slsa}{nbsp}12 channels appropriate for your architecture, including the Live Patching child channel (or channels)
+* The clients are fully synchronized
+
+.Procedure: Setting up for Live Patching
+
+. Select the client you want to manage with Live Patching from menu:Systems[Overview], and navigate to the menu:Software[Packages > Install] tab.
+Search for the [systemitem]``kgraft`` package, and install it.
++
+image::enable_live_patching_kgraft_install.png[scaledwidth=80%]
+. Apply the highstate to enable Live Patching.
+. Repeat for each client that you want to manage with Live Patching.
+. To check that Live Patching has been enabled correctly, select the client from menu:Systems[Systems List], and ensure that [systemitem]``Live Patching`` appears in the [guimenu]``Kernel`` field.
```

Comment:
IIRC, systems will need a reboot when live patching is enabled for the first time. I need to test it out in both versions to verify. If that is correct, we should mention it here.

I'll get back to you after I try it out ASAP.

Reply:
@cbbayburt Any update here?

Reply:
@Loquacity, I can confirm for both versions that a reboot is required after the **initial** install of a live patching package.
In the UI Overview page, you can see both the live patching version and the reboot required text.